roles/pre-ansimail/files/ansimail_scripts/add-user

```diff
@@ -31,12 +31,12 @@ case $user_status in
 esac
 
 printf "Password: "
-read password
-echo "\n"
+read -r password
```

Comment:
i didn't even know this was a thing :O

Reply:
> i didn't even know this was a thing :O

It stops \'s from being mangled on input.